components/checkbox/stories/template.js

```diff
@@ -62,13 +62,19 @@ export const Template = ({
 				type="checkbox"
 				class="${rootClass}-input"
 				aria-labelledby=${ifDefined(ariaLabelledby)}
+				aria-disabled=${ifDefined(isReadOnly ? "true" : undefined)}
```

Comment:
I can look it up on MDN, but I wonder why `aria-disabled` adds the "dimmed" part of the screen reader announcement, as opposed to just saying something like "read-only." 🤔